R-package/R/plot_results.r

```diff
@@ -3,17 +3,16 @@
 #'A quick wrapper to produce a habitat type map and habitat type proportions in a pie chart.
 #'
 #' @param inPath file path (character) for results of habitat type sampling and probability mapping (same as outPath from function multi_Class_Sampling)
+#' @param threshold probability threshold provided by the user.
```

Comment:
the threshold should be loaded automatically from the file stored at disk that is generated as output from the multi_Class_Sampling funtion. Why is it integrated here as argument?

Reply:
Because the when we are inside a function, we should not use global variables. As you can see in [plot_results.r#L36]*(https://github.com/carstennh/HabitatSampler/blob/master/R-package/R/plot_results.r#L36) `threshold` is used without ever being initialized (I can see that you set in the `outer_procedure.r`, but we can't assume that will be done once we call the function in another location). Since initialization is thus dependent on something else, it is better to pass it as argument.

Let me know if the description is correct so I can improve it.

Reply:
in the line before 36, the variable threshold is loaded from disk...since we specify the inPath as function argument, threshold should always be loaded as part of the name convention that is generated in outer_procedure.r Is that ok?

Reply:
We can leave like that, however, it is a bit dangerous this mode. In case you you have a variable with the same name, the load will change overwrite it. For single objects the best is to use `saveRDS`, it saves an object to a single file and the user can reload it and assign it to a variable. 

In both cases, when we save objects using `save` and `saveRDS` we should use file extensions as well, like `rda` and `rds`.

I will revert my change and add a comment above line 35 so users understand from where it comes.

Reply:
Covered by 44ef76d.